lib/buy/moonpay/moonpay_provider.dart

```diff
@@ -235,72 +320,79 @@ class MoonPayProvider extends BuyProvider {
         walletId: wallet.id);
   }
 
-  static Future<bool> onEnabled() async {
-    final url = _apiUrl + _ipAddressSuffix + '?apiKey=' + _apiKey;
-    var isBuyEnable = false;
-    final uri = Uri.parse(url);
-    final response = await get(uri);
-
-    try {
-      final responseJSON = json.decode(response.body) as Map<String, dynamic>;
-      isBuyEnable = responseJSON['isBuyAllowed'] as bool;
-    } catch (e) {
-      isBuyEnable = false;
-      print(e.toString());
+  String _normalizeCurrency(CryptoCurrency currency) {
```

Comment:
this is not used, do we not need it anymore?

Reply:
still need to use it in a specific case, fixed